### PR TITLE
Update getting started docs and fix documentation layout

### PR DESCRIPTION
**Pull request by tatang26**
I updated the following getting started docs:

Install Buffalo:
- I removed unneded `$GOPATH` information

Generating a New Project:
- I removed unneded `$GOPATH` information
- I updated the `buffalo new coke` output information
- I updated the `buffalo help new` output information
- I removed the `Configuration` next step link

Directory structure:
- I updated the structure of a Buffalo project
- I added a little `cmd` information.

Configuration
- I added the `Next Steps` section

Tooling Integration
- I added information about go extension for visual studio code

Besides, I adjusted the documentation layout in order to fill the content into box

### CODE REVIEW

File: assets/css/home.css [hunk 1]
```diff
@@ -96,7 +92,7 @@ html {
     @apply p-3 overflow-scroll;
 }
 
-.content p code, li code {
+.content p code, li code, table code {
```
Comment (sio4) [inline assets/css/home.css:95]: just my personal preference, gray background inside the table doesn't look good to me :-)

**Review by sio4 — COMMENTED** [review]
Just left one comment. I am open to it but just wanted to tell you my impression :-)

Otherwise, looks great!

File: assets/css/home.css (hunk 1 above)
Comment (sio4) [reply assets/css/home.css:95]: yeah, it is just my personal preference and just wanted to know the other's thoughts. Anyway, I am fine, and thank you for your nice work!

**Review by sio4 — APPROVED** [review]
LGTM!

**Comment by paganotoni** [comment]
This is Great! Thanks @tatang26

**Pull request merged**